docker-compose.yml

```diff
@@ -0,0 +1,29 @@
+version: "3.9"
+    
+services:
+  waltz:
+    image: ghcr.io/ljubon/waltz:postgresql 
+    restart: always
+    command: ["update","run"]
```

Comment:
I would remove this as it's the default. Our goal should be to have the simplest possible `docker-compose.yml` file. Advanced users can customise based on the README.